torch/_inductor/select_algorithm.py

```diff
@@ -447,12 +448,32 @@ def make_kernel_render(out_node):
             )
             return kernel, render
 
+        # create the BenchmarkRequest if we do autotuning in subprocess
+        bmreq = None
```

Comment:
Can we use a `BenchmarkRequest` object for both subproc and nonsubproc to reuse more code?

Reply:
yep, will consolidate the code for these 2 cases.